src/main/java/chatbot/Ui.java

```diff
@@ -0,0 +1,67 @@
+package chatbot;
+import java.util.Scanner;
+
+/*
+ * The Ui class handles the interactions with the user.
+ * It provides methods to read input and display formatted messages.
+ */
+public class Ui {
+  private Scanner scanner;
+
+  /*
+   * Constructor to create Ui object and initializes Scanner.
+   */
+  public Ui() {
+    this.scanner = new Scanner(System.in);
+  }
+
+  /*
+   * Reads a single line of input from the user.
+   * 
+   * @return The inputted string.
+   */
+  public String readLine() {
+    return scanner.nextLine();
+  }
+
+  /*
+   * Displays welcome message.
+   */
+  public void welcomeMessage() {
+    System.out.println("____________________________________________________________\r\n" + //
+            " Hello! I'm Ervin Chatbot!\r\n" + //
+            " What can I do for you?\r\n" + //
+            "____________________________________________________________\r\n");
+  }
```

Comment:
Consider abstracting this print statement into a separate method such as `printDivider()` for better readability and reusability.

```suggestion
  /*
   * Displays welcome message.
   */
  public void welcomeMessage() {
    printDivider();
    System.out.println(" Hello! I'm Ervin Chatbot!\r\n What can I do for you?\r\n");
    printDivider();
  }
```
```java
private void printDivider() {
    System.out.println("____________________________________________________________\r\n"

}
```